src/xSTIR/pSTIR/STIR.py

```diff
@@ -1476,12 +1476,25 @@ def set_acquisition_sensitivity(self, asm):
         # save reference to the Acquisition Sensitivity Model
         self.asm = asm
 
-    def forward(self, image, subset_num=0, num_subsets=1, out=None):
+    def forward(self, image, subset_num=None, num_subsets=None, out=None):
         """Return the forward projection of image.
 
-        image   :  an ImageData object.
+        image      :  an ImageData object.
+        subset_num : optional (int) subset number to forward project. 
+                     Default None. If None subset_num is the instance 
+                     property subset_num.
+        num_subsets: optional (int) number of subsets the data is divided 
+                     in for the projection. Default None. 
+                     If None num_subsets is the instance property num_subsets.
```

Comment:
can we formulate this in a less Pythonic-way? Most of our audience will not understand "instance property"

Reply:
I understand this request but I believe it's better to be clear and right on the programming point of view. I could add "member" in parenthesis.

Reply:
you could write e.g. "the number of subsets is set to `self.num_subsets`"